autode/conformers/conformers.py

```diff
@@ -19,6 +20,23 @@ def _calc_conformer(conformer, calc_type, method, keywords, n_cores=1):
 
 class Conformers(list):
 
+    @property
+    def lowest_energy(self) -> Optional['autode.conformers.Conformer']:
+        """
+        Return the lowest energy conformer state from this set. If no
+        conformers have an energy then return None
+
+        -----------------------------------------------------------------------
+        Returns:
+            (autode.conformers.Conformer | None): Conformer
+        """
+        if all(c.energy is None for c in self):
+            logger.error('Have no conformers with an energy, so no lowest')
+            return None
+
+        energies = [c.energy if c.energy is not None else np.inf for c in self]
+        return self[np.argmin(energies)]
```

Comment:
I think [np.nanargmin](https://numpy.org/doc/stable/reference/generated/numpy.nanargmin.html) does this entire function for you, if you wrap in a try/except

Reply:
actually thats nans, so possibly ignore me...

Reply:
I try 😉. Also:

> Warning: the results cannot be trusted if a slice contains only NaNs and Infs.

lol